cristin-person/src/test/java/no/unit/nva/cristin/person/handler/person.feature

```diff
@@ -158,3 +158,12 @@ Feature: API tests for Cristin Person fetch
     * string identifiers = response.identifiers
     And match identifiers contains 'CristinIdentifier'
     And match identifiers !contains 'NationalIdentificationNumber'
+
+  Scenario: Get returns supported fields in payload
+    * def samplePersonIdentifier = '515114'
+    Given path '/person/' + samplePersonIdentifier
+    When method GET
+    Then status 200
+    And match response == '#object'
+    And match response.verified == '#present'
+    And match response.verified == true
```

Comment:
Question, does response.verified == true  return false if verified is missing or undefined? 
Or does it crash?

Reply:
It must be there, if missing it will fail the test. But this is a test person which would not change status verified